mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -1853,6 +1857,85 @@ def instructions(self) -> str:
         return CONVERSATIONAL_SAFETY_PROMPT
 
 
+@experimental(version="3.8.0")
+@format_docstring(_MODEL_API_DOC)
+class ConversationalToolCallEfficiency(BuiltInSessionLevelScorer):
+    """
+    Conversational tool call efficiency evaluates whether tool usage across a
+    multi-turn conversation session was optimized.
+
+    This scorer analyzes the complete conversation and tool call history for efficiency:
```

Comment:
super nit: I like being descriptive about the metric, but maybe let's make this more generic to avoid leaking implementation details in the docstring. This is important if we ever change the prompt (e.g., remove one of these checks), so we don't have to also think about updating this docstring.